cli/ioctl/cmd/account/accountcreate.go

```diff
@@ -46,11 +56,22 @@ func accountCreate(_ []string) string {
 			log.L().Error("failed to convert bytes into address", zap.Error(err))
 			return err.Error()
 		}
-		priKeyBytes := private.Bytes()
-		pubKeyBytes := private.PublicKey().Bytes()
-		items[i] = fmt.Sprintf(
-			"{address: \"%s\", privateKey: \"%x\", publicKey: \"%x\"}",
-			addr.String(), priKeyBytes, pubKeyBytes)
+		newAccount := generatedAccount{
+			Address:    addr.String(),
+			PrivateKey: fmt.Sprintf("%x", private.Bytes()),
+			PublicKey:  fmt.Sprintf("%x", private.PublicKey().Bytes()),
+		}
+		newAccounts = append(newAccounts, newAccount)
+	}
+	var output []byte
+	var err error
+	if numAccounts == 1 {
```

Comment:
no need for `if numAccounts == 1`, `json.Marshal(&generatedAccounts{Accounts: newAccounts})` would work both cases

Reply:
This case is to make print more human readable, I get a better solution now.